# This is a time message including a sequence number for association with a payload message.
#

time time
uint64 number
uint64 exposure_pri
uint64 exposure_sec

